debug_toolbar/static/debug_toolbar/js/utils.js

```diff
@@ -91,6 +106,14 @@ function ajax(url, init) {
         });
 }
 
+function pluckData(array, key) {
```

Comment:
This function can likely be replaced with:

```javascript
return array.map(function(obj) {
    return obj.dataset[key]
})
```

Reply:
That was my first attempt, but annoyingly Set objects don't have a map function.